docs/extend/extend-desktop/mvd-server-config.md

```diff
@@ -1,13 +1,12 @@
 The Zowe's App Server and ZSS rely on many required or optional parameters to run, which includes setting up networking, deployment directories, plugin locations, and more. 
-These parameters can be specified in multiple ways: configuration files, CLI arguments, or environment variables.
+These parameters can be specified in two ways: configuration files, or environment variables.
```

Comment:
Is this md file supposed to be in Zowe Docs? I ask b/c it's not included in the sidebar file, therefore it doesn't appear on the Zowe Docs site (although it _is_ in the Zowe Docs repository). If this is supposed to be included in the site, please add the file to the appropriate location in the `sidebar.js` file.